docs/dev/tags.rst

```diff
@@ -1,27 +1,25 @@
 
 
-Basic process of adding and using a new tag:
+Basic process of creating and using a new tag:
 --------------------------------------------
 
 To create a *global tag* (one applies to all programs), add a new line to the global tag dictionary ``all_global_tags`` in the
 file https://github.com/learning-unlimited/ESP-Website/blob/main/esp/esp/tagdict/__init__.py.
 To create a *program-specific tag*, add the new tag's name to the program tag dictionary ``all_program_tags`` in the same file as above.
-Program tags are used with ``Tag.getProgramTag()`` or ``Tag.getBooleanTag()`` with a program argument in place of a boolean one in the
-first coordinate.
 
-The format for tags in either dictionary is ``'tag_name': (is the tag boolean?, 'some help text for admins')``, where the first
-coordinate in the tuple determines whether the tag is retrieved by the function ``Tag.getBooleanTag`` (if so, set the first coordinate
-to ``True``) or ``Tag.getTag`` (set it to ``False``). This formatting information is also in the comments of the file linked above.
+The format for tags in both dictionaries is ``'tag_name': (is the tag boolean?, 'some help text for admins')``, where the first
+variable in the tuple determines whether the tag is meant to contain only a boolean value (if so, set the first variable to ``True``) or other data (set it to ``False``).
 
 In order to make use of your new tag, find the file(s) where you want to use the tag's data, and make sure to include the
 line ``from esp.tagdict.models import Tag``.
-Use ``Tag.getTag('tag_name')`` (or the apprpriate variant, either ``getBooleanTag`` or ``getProgramTag``) to retrieve the value of the
-tag in each file.
+Use ``Tag.getTag('tag_name')`` to retrieve the value of a non-boolean global Tag.
+For boolean Tags, use ``Tag.getBooleanTag('tag_name')``.
+Program tags can be used with ``Tag.getProgramTag('tag_name', program)`` or by passing a program argument to ``Tag.getBooleanTag()``.
 
 
 *Notes:*
-Due to the import line, it is not possible to use tag data in `models\__init__.py`.
-You may want to implement checks against the validity of the tag's data.
-It is especially helpful to send a helpful error message to either the admin and the server log. :)
 
-Please also add a description of your tag to the LU Wiki page on tags.
+* Due to the import line, it is not possible to retrieve tag data at the top-level of any ``models`` file or any other file that a ``models`` file imports.
```

Comment:
The new explanation still doesn't make sense to me, although the instruction itself is correct now. The reason for this error is that you [can't use the database until all models have finished loading](https://docs.djangoproject.com/en/1.8/ref/applications/#troubleshooting), and it would probably be a bad idea anyway, as Ben said. I think I'd express the underlying rule here as something like "Because tag data is stored in the database, it shouldn't (and in some cases can't) be accessed at import time."